src/backend/vulkan/src/physical_device.rs

```diff
@@ -191,54 +702,14 @@ impl adapter::PhysicalDevice<Backend> for PhysicalDevice {
             return Err(CreationError::MissingFeature);
         }
 
-        let imageless_framebuffers = self.api_version >= Version::V1_2
```

Comment:
Inlined in the `Device` initializer below--we no longer need this exact condition since the inclusion of `PhysicalDeviceVulkan12Features` makes the inclusion of imageless framebuffers slightly more complex